src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/AssemblyAttributeProperties/AssemblyInfoProperties.cs

```diff
@@ -99,12 +99,15 @@ private async Task<string> GetPropertyValueFromSourceAttributeAsync(string prope
         /// <summary>
         /// Set the value of a property.
         /// </summary>
-        public override async Task SetPropertyValueAsync(string propertyName, string unevaluatedPropertyValue, IReadOnlyDictionary<string, string> dimensionalConditions = null)
+        public override async Task SetPropertyValueAsync(string propertyName, string? unevaluatedPropertyValue, IReadOnlyDictionary<string, string>? dimensionalConditions = null)
         {
             if (_attributeValueProviderMap.TryGetValue(propertyName, out SourceAssemblyAttributePropertyValueProvider provider) &&
                 !await IsAssemblyInfoPropertyGeneratedByBuild(propertyName))
             {
+#pragma warning disable CS8604
                 await provider.SetPropertyValueAsync(unevaluatedPropertyValue);
```

Comment:
Please review - not sure if we should be allowing nulls here, and if not, what we should do?  Throw/skip the call/etc?